cfme/fixtures/ansible_tower.py

```diff
@@ -124,3 +127,30 @@ def ansible_tower_dialog(request, appliance):
 
     if appliance.version < '5.11' or not GH(8836).blocks:
         service_dialog.delete_if_exists()
+
+
+@pytest.fixture
+def ansible_api_version_change(provider, ansible_api_version):
+    """Fixture to update Tower url to /api/v2 so that API v2 can be tested.
+
+    API version defaults to v1. So, if no version is specified, v1 is used except for things
+      which don't exist on v1.
+    If v2 is specified, v2 is used for everything.
+
+    Ansible Tower 3.4, 3.5 support both API v1 and v2.
+    API v1 has been fully deprecated in Ansible Tower 3.6 and Tower 3.6 supports API v2 only.
+
+    """
+    parsed = urlparse(provider.url)
+
+    if ansible_api_version == 'v2':
+        updated_url = f'{parsed.scheme}://{parsed.netloc}/api/{ansible_api_version}'
+        original_url = provider.url
```

Comment:
Looking good, but I would do two things here:

1. move this assignment of original_url outside of the logic blocks, so that we don't hit any UnboundLocals
2. Make the url update regardless of `ansible_api_version`, so that its always explicitly set to what the test parameter is.

If we're not explicitly setting it, we risk running into situations where fixture/parameter scope, or the URL defined for the provider in the yaml, will result in the provider's configuration not matching the parametrized value for API.

This way we can remove the logic here.

Reply:
Done